apicast-gateway/apicast.yml

```diff
@@ -82,6 +82,13 @@ objects:
           - name: management
             containerPort: 8090
             protocol: TCP
+          resources:
+          limits:
+              cpu: '1'
+              memory: 512Mi
```

Comment:
I think it could easily live with a half of that. Even with 128M.

Reply:
128 ~150Mb , depends on the number of apicast workers.. 

And I don't see any change to the default APICAST_WORKERS...

Reply:
@jmprusi can you review this PR? 

We have the APICAST_WORKERS in place now and QE have already tested it